presto-main/src/main/java/com/facebook/presto/sql/planner/plan/AssignmentUtils.java

```diff
@@ -41,6 +41,16 @@ public static Assignments identityAssignments(VariableReferenceExpression... var
         return identityAssignments(asList(variables));
     }
 
+    public static boolean isIdentity(Assignments assignments)
```

Comment:
we have very similar logic in ProjectNodeUtils.isIdentity().  Can we combine them so we don't duplicate? (maybe have that call AssignmentUtils.isIdentity()?)